CMake/ConfigureCrossCompile.cmake

```diff
@@ -5,10 +5,10 @@
 # available on your system in order to find the BLAS library.  If OpenBLAS will
 # be compiled, the OPENBLAS_TARGET variable must be set.  This can be done
 # by, e.g., setting BOARD_NAME (which will set OPENBLAS_TARGET in
-# `board/flags-config.cmake`).
+# `flags-config.cmake`).
 
 if (CMAKE_CROSSCOMPILING)
-  include(board/flags-config.cmake)
+  include(CMake/flags-config.cmake)
```

Comment:
Since everything is in `CMake/` now, the `flags-config.cmake` filename could be a little bit confusing---it's not clearly about the cross-compilation flags.  Maybe it's worth moving to `cross-compile-config.cmake` or some name like that?